### PR TITLE
[IMP] accounting: cash register page in POS

### PR DESCRIPTION
**Pull request by afma-odoo**
task-3721929
task-3890538

### CODE REVIEW

File: content/applications/sales/point_of_sale.rst [hunk 1]
```diff
@@ -118,6 +118,17 @@ To return and refund a product,
      setting a negative quantity that equals the number of returned products. To do so, click
      :guilabel:`Qty` and :guilabel:`+/-`, followed by the quantity of returned products.
 
+.. _pos/cash-register:
+
+Manage the cash register
+========================
+
+To add or take out cash from the register, click :guilabel:`Cash In/Out` in the upper left side of
+your screen.
```
Comment (auva-odoo) [inline content/applications/sales/point_of_sale.rst:127]:
```suggestion
To add or take out cash to/from the register, click :guilabel:`Cash In/Out` on the upper left side of
the screen.
```

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Just a small suggestion @afma-odoo :slightly_smiling_face:

File: content/applications/sales/point_of_sale.rst (hunk 1 above)
Comment (afma-odoo) [reply content/applications/sales/point_of_sale.rst]: @auva-odoo Thanks for the advice 😊

**Review by auva-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**